.github/workflows/release_tag.yml

```diff
@@ -26,11 +25,20 @@ jobs:
           OUT=$(jq --slurp '[.[].packages[] | select(.name | contains("ezsystems") or contains("ibexa")) | [.name, .version] | { name: (.[0]), version: .[1] }]' composer.lock)
           echo "::set-output name=lock::$( echo "$OUT" | sed ':a;N;$!ba;s/\n/%0A/g' )"
 
+      - name: Get previous last full release
+        id: prevfull
+        run: |
+          OUT=$(hub api /repos/${{ github.repository }}/releases | jq -r -s '[ .[][].tag_name | select(. | contains("rc") or contains("beta") or contains("alpha") | not) ] | first')
+          echo "::set-output name=tag::$( echo "$OUT" | sed ':a;N;$!ba;s/\n/%0A/g' )"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Get previous release tag based on type
         id: prevrelease
         uses: ibexa/version-logic-action@master
         with:
           currentTag: ${{ env.BUILD_TAG }}
+          prevFullTag: ${{ steps.prevfull.outputs.tag }}
```

Comment:
Given the input `v3.3.9`, action output is expected to be `v3.3.0`.

Considering [tests have passed](https://github.com/ibexa/version-logic-action/runs/3765205267?check_suite_focus=true), specifically the `alphatest1` and [corresponding assert](https://github.com/ibexa/version-logic-action/actions/runs/1294367098/workflow#L56) are ok - this should be fine.